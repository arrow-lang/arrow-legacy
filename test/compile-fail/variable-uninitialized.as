let value;
let value = value;  // error
