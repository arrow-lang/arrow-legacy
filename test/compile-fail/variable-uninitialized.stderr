[0;37mtest/compile-fail/variable-uninitialized.as:2.13-18: [0;31merror: [1;37muse of possibly uninitialized variable 'value'[0m
